packages/react/src/primitives/Select/Select.tsx

```diff
@@ -57,6 +59,8 @@ const SelectPrimitive: Primitive<SelectProps, 'select'> = (
             : defaultValue
         }
         isDisabled={isFieldsetDisabled ? isFieldsetDisabled : isDisabled}
+        multiple={isMultiple}
+        size={selectSize || 1}
```

Comment:
```suggestion
        size={selectSize}
```

Since you added a default value you won't need this `||`